recipe/meta.yaml

```diff
@@ -17,20 +18,24 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - pytest-runner
+    - wheel
     - setuptools >=30.0.0
   run:
     - python >=3.6
     - mccabe >=0.6.0,<0.7.0
-    - pycodestyle >=2.7.0,<2.8.0
-    - pyflakes >=2.3.0,<2.4.0
-    - importlib-metadata  # [py<38]
-    - setuptools >=30.0.0
+    - pycodestyle
+    - pyflakes
+    - importlib-metadata
+    - setuptools
```

Comment:
the pinnings for `importlib-metadata` and `setuptools` were corrected

Reply:
`setuptools` isn't mentioned in the upstream source as install_requires (it equal the run section in the conda recipe). It is need only in host